arrow-fx-coroutines/src/main/kotlin/arrow/fx/coroutines/CircuitBreaker.kt

```diff
@@ -5,6 +5,9 @@ import arrow.core.identity
 import arrow.fx.coroutines.CircuitBreaker.State.Closed
 import arrow.fx.coroutines.CircuitBreaker.State.HalfOpen
 import arrow.fx.coroutines.CircuitBreaker.State.Open
+import kotlin.time.Duration
+import kotlin.time.milliseconds
+import kotlin.time.nanoseconds
 
 class CircuitBreaker constructor(
```

Comment:
The new constructor should be `private`, so people are forced to use the `of` constructor.
We should also deprecate the original constructor like we did for the `ADT`, but point to the `of` method with `ReplaceWith` instead of this new one.

```suggestion
class CircuitBreaker private constructor(
```